docs: add note for tests that have requisites (#6856)
excalidraw/excalidraw: dev-docs/docs/introduction/contributing.mdx

Finally - always manually test your changes using the convenient staging environment deployed for each pull request. As much as local development attempts to replicate production, there can still be subtle differences in behavior. For larger features consider testing your change in multiple browsers as well.

:::note

> Some tests require aproval from the maintainers to run. 
> These tests are the `lint`, `semantic` and `test` tests.
> They will appear as `Expected — Waiting for status to be reported` in the PR checks when they are waiting for approval.

Reviewer: ```suggestion
Some checks, such as the `lint` and `test`, require approval from the maintainers to run. 
They will appear as `Expected — Waiting for status to be reported` in the PR checks when they are waiting for approval.
```
